dotcom-rendering/src/components/SubMeta.tsx

```diff
@@ -184,7 +198,7 @@ export const SubMeta = ({
 						/>
 					</Island>
 					<div css={syndicationButtonOverrides}>
-						{format.design === ArticleDesign.Interactive ? null : (
+						{showSindicationButton ? (
```

Comment:
```suggestion
						{showSyndicationButton ? (
```

Reply:
Commit for this https://github.com/guardian/dotcom-rendering/pull/14137/commits/a2ecd44ae2c4c2144d4c7aa948453a29e2cd8e4b